variables/_variables.breakpoints.scss

```diff
@@ -0,0 +1,9 @@
+  // These variables are the default AP variables and can be overridden in other brand kits
```

Comment:
Plz can you move this dev note into the Variables section of the readme?

Reply:
done